### DISCUSSION TITLE
根据官方流程制作增值税发票数据集和教程不符

### DISCUSSION THREAD

**Discussion by whisky-12** [PaddlePaddle/PaddleOCR#12866] [Q&A]
使用PPOCRLabelV2 根据教程标注制作后的数据集 和 官方教程中的数据集格式不符 

官方数据集
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79358023/bc39870c-d037-4761-a747-aec202bcced7)

PPOCRLabelV2 标注后的得到的

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79358023/3b0cb94e-b004-437d-923b-77d5dc52e11e)

制造过程细节

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79358023/30176b86-cc62-40f1-be0d-1612bc811de8)

执行python gen_ocr_train_val_test.py --trainValTestRatio 8:1:1 --datasetRootPath ../train_data/imgs
对数据集划分后 ：
数据集分为det 和 rec ，和官方发票数据集格式不符 ，请问官方数据集是如何制作的 ，json 文件中 label 标签如何标注绑定

**Comment by whisky-12** [comment]
请问各位伙伴 有知道  
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppstructure/kie/how_to_do_kie.md
中·提到的 连接信息 是用什么标注软件进行标注的
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79358023/0972f08f-1d8f-4347-b50a-d8fc1f726009)

**Comment by Rishi-NF** [comment]
No one wants to answer this question lol

**Comment by Rishi-NF** [comment]
This might help 

[https://gitee.com/paddlepaddle/PaddleOCR/blob/release/2.6/applications/%E5%8F%91%E7%A5%A8%E5%85%B3%E9%94%AE%E4%BF%A1%E6%81%AF%E6%8A%BD%E5%8F%96.md#431-%E5%87%86%E5%A4%87%E6%95%B0%E6%8D%AE](url)

[https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_en/data_annotation_en.md](url)

**Comment by zjlw** [comment]
有解决方法了吗？

**Comment by whisky-12** [comment]
目前没有遇到对应的解决办法 ，目前使用的标注软件都没能做到官方效果的格式，参考了Studio中一个开源项目， 对通用标注后的标注结果处理 ，但也没能实现
